src/commands/config/create.ts

```diff
@@ -20,7 +20,13 @@ import tradetrustConfig from "./templates/tradetrust.json";
 import { withWalletOption } from "../shared";
 
 interface ConfigFile {
-  wallet: string;
+  wallet: {
```

Comment:
Let's split the declaration. One for AWS with expected keys and one for encrypted with the expected keys

Reply:
ok